writing/2016/tacl_colors.tex

```diff
@@ -585,6 +585,7 @@ \section{Models}
 
 Finally, a \term{pragmatic listener} (\figref{fig:basic-rsa:l2})
 interprets utterances by reasoning about the behavior of the pragmatic speaker:
+\textcolor{red}{rdh: recent RSA papers have switched to using $l_1$ as notation for the pragmatic listener that reasons about $s_1$ instead of going up a number with every recursion, although I can also see how it's useful to distinguish between this $l_1$ and the $l_1$ that reasons about a literal speaker later\dots}
```

Comment:
Yeah, I agree and can't think of anything better that doesn't involve cumbersome subscript... Maybe it'd just be worth flagging the notation difference in a footnote, just in case the reader is familiar with RSA and gets confused?